src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,130 @@
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ClimberConstants;
+import frc.robot.Constants.MotorIDConstants;
+import frc.robot.Robot;
+import java.util.function.BooleanSupplier;
+
+public class Climber extends SubsystemBase {
+  /** Creates a new Climber. */
+  private TalonFX climberMotorFront;
+
+  private TalonFX climberMotorBack;
+  private DCMotor simClimberGearbox;
+  private SingleJointedArmSim climberSim;
+  private Mechanism2d climbMech;
+  private MechanismRoot2d climbMechRoot;
+  private MechanismLigament2d climbMechLigament;
+  private Slot0Configs climberConfigsToClimber;
+  private Slot1Configs climberConfigsAtClimber;
+
+  public Climber() {
+    climberMotorFront = new TalonFX(MotorIDConstants.kCLimberMotorLeader);
+    climberMotorBack = new TalonFX(MotorIDConstants.kCLimberMotorFollower);
+    climberConfigsToClimber =
+        new Slot0Configs()
+            .withKP(ClimberConstants.kClimberP0)
+            .withKI(ClimberConstants.kClimberI0)
+            .withKD(ClimberConstants.kClimberD0)
+            .withKG(ClimberConstants.kClimberkG0)
+            .withKV(ClimberConstants.kClimberkV0);
+    climberConfigsAtClimber =
+        new Slot1Configs()
+            .withKP(ClimberConstants.kClimberP1)
+            .withKI(ClimberConstants.kClimberI1)
+            .withKD(ClimberConstants.kClimberD1)
+            .withKG(ClimberConstants.kClimberkG1)
+            .withKV(ClimberConstants.kClimberkV1);
+
+    if (Robot.isSimulation()) {
+      simClimberGearbox = DCMotor.getKrakenX60(1);
+      climberSim =
+          new SingleJointedArmSim(
+              simClimberGearbox,
+              ClimberConstants.KGearRatio,
+              ClimberConstants.ClimberSimConstants.kClimberArmMOI.in(KilogramSquareMeters),
+              ClimberConstants.ClimberSimConstants.kClimberArmLength.in(Meters),
+              0,
+              Math.PI,
+              false,
+              1);
+      climbMech = new Mechanism2d(2, 2);
+      climbMechRoot = climbMech.getRoot("Climb Mech root", 1, 1);
+      climbMechLigament =
+          climbMechRoot.append(
+              new MechanismLigament2d(
+                  "Climb Mech Ligament", 1, Radians.of(climberSim.getAngleRads()).in(Degrees)));
+      SmartDashboard.putData("Climb Mech", climbMech);
+    }
+  }
+
+  private Command runClimber(Angle position, int slot) {
+    return runOnce(
+        () -> {
+          climberMotorFront.setControl(
+              new PositionDutyCycle(position.times(ClimberConstants.KGearRatio)).withSlot(slot));
+          climberMotorBack.setControl(
+              new PositionDutyCycle(position.times(ClimberConstants.KGearRatio * -1))
```

Comment:
Why are you multiplying by -1 instead of making it negetive?
Also I think you should make the back motor follow the front motor. This is a better way of doing this.